### DISCUSSION TITLE
Feature Request: Automatically Update Yt-dlp when app launches

### DISCUSSION THREAD

**Discussion by trymeouteh** [JunkFood02/Seal#677] [Ideas]
Please add a setting to have Seal check for updates to Yt-dlp when the user opens the app and if there is an newer version of Yt-dlp, to update Yt-dlp to the latest version.

**Comment by Marco-9456** [comment]
https://user-images.githubusercontent.com/41828058/226141404-9cc6ad31-a9d3-4f7b-a4bb-6e6bdd772307.mp4

**Comment by trymeouteh** [comment]
When will 1.9.0 be released for F-Droid? F-Droid still has 1.8.3
https://f-droid.org/en/packages/com.junkfood.seal/

**Reply by Marco-9456** [reply]
> When will 1.9.0 be released for F-Droid? F-Droid still has 1.8.3 https://f-droid.org/en/packages/com.junkfood.seal/

F-Droid releases have usually a 5-6 days delay, and this is something we can't control, F-Droid build the app on it's own and use a different signature, i recommend switching to GitHub [releases](https://github.com/JunkFood02/Seal/releases) and enable Auto update, this way you'll get new releases directly after its published on GitHub.

**Reply by trymeouteh** [reply]
Will 1.9.0 ever be released on F-droid?

**Reply by JunkFood02** [reply]
> Will 1.9.0 ever be released on F-droid?

F-Droid server is having issues on updates which is not in our control

**Comment by HassanDalwi** [comment]
Where is the trim option on 1.9.0 is gone

**Reply by Marco-9456** [reply]
> Where is the trim option on 1.9.0 is gone

Settings --> Format --> Clip video.
